<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatDialog</class>
 <widget class="QWidget" name="chatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="tb_chatRecord">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_send">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="le_insert">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>401</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_openFile">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="sca_fileTran">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>0</y>
     <width>501</width>
     <height>360</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>499</width>
      <height>358</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tb_fileInfo">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <column>
        <property name="text">
         <string>文件名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>上传/下载</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>进度</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>openFile</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
